src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -1674,7 +1664,7 @@ public Duration RTT() throws IOException {
         pongQueue.add(pongFuture);
         try {
             long time = NatsSystemClock.nanoTime();
-            writer.queueInternalMessage(new ProtocolMessage(PING_PROTO));
+            writer.queue(new ProtocolMessage(PING_PROTO));

```

Comment:
This was changed after a review of the internal queuing and determined it was kinda pointless. If they make an rtt call while normally connected, this message just goes into the normal queue anyway. If they made the rtt call during reconnect, it would have gone into the internal queue, but so what, it's time is going to be skewed anyway.
This whole RTT may need to be put into a "bypass" queue of some sort so on normal connect, it goes directly to the front of the line.